docs/user/tutorials/01-30-expose-tcp-gateway-api.md

```diff
@@ -0,0 +1,129 @@
+# Use Gateway API to Expose HTTPBin
+
+This tutorial shows how to expose an HTTPBin Service using Gateway API.
+
+> [!WARNING]
+> This tutorial is based on experimental Istio module.
+> Exposing a workload to the outside world is a potential security vulnerability, so tread carefully. This example is not meant to be used in production environment. 
+
+## Prerequisites
+
+* Experimental Istio module installation
+
+## Steps
+
+### Configure Gateway API alpha support
+
+Edit Istio CR by setting `enableAlphaGatewayAPI` to true. Your Istio CR should look:
+
+    ```
+    ...
+    spec:
+      config: {}
+      experimental:
+        pilot:
+          enableAlphaGatewayAPI: true
+          enableMultiNetworkDiscoverGatewayAPI: false
+    ...
+    ```
+
+### Install experimental version of Gateway API CRDs
+
+Istio module does not install Gateway API CRDs. To install CRDs execute following command:
+
+    ```
+    kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
```

Comment:
If you first do the HTTP tutorial and then the TCP tutorial it will not work, since in this step, the experimental CRDs will not be applied as `kubectl get crd gateways.gateway.networking.k8s.io` will return the CRD created in the previous step.
I don't know if we want to make it bullet-proof by first deleting existing gateway CRDs before applying them.

Reply:
these are 2 independent tutorials. User should know what CRDs they require and install them.